MauiProgram.cs

```diff
@@ -28,7 +28,8 @@ public static MauiApp CreateMauiApp()
             Domain = "<YOUR_AUTH0_DOMAIN>",
             ClientId = "<YOUR_CLIENT_ID>",
             Scope = "openid profile",
-            RedirectUri = "myapp://callback/"
+            RedirectUri = "myapp://callback/",
+            PostLogoutRedirectUri = "myapp://callback"

```

Comment:
For consistency with the updated article, it should be as follows:

```suggestion
            PostLogoutRedirectUri = "myapp://callback/"

```